test/suite/wttest.py

```diff
@@ -221,6 +221,11 @@ def globalSetup(preserveFiles = False, useTimestamp = False,
         WiredTigerTestCase._concurrent = False
         WiredTigerTestCase._globalSetup = True
         WiredTigerTestCase._ttyDescriptor = None
+        WiredTigerTestCase._seeds = [521288629, 362436069]
```

Comment:
I like that your changes have the "standard" random seeds in only one occurrence now.

Reply:
thank you!